fiftyone/core/utils.py

```
@@ -3128,6 +3128,76 @@ def get_cpu_count():
+            pass
+
+        cgroup = None
+        if sys.platform.startswith("linux"):
+            # cgroup v2
+            try:
+                with open("/sys/fs/cgroup/memory.max") as f:
+                    value = f.read().strip()
+                    if value != "max":
+                        cgroup = int(value)
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)